### PR TITLE
docs: split loader docs

### PR DESCRIPTION
**Pull request by SyMind**
## Summary

image

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/api/loader-api/context.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+## `this.getOptions(schema)`
+
+Extracts the given loader option, accepting an optional JSON schema as an argument.
+
+## `this.getResolve(options: ResolveOptions): resolve`
+
+Create a resolver like `this.resolve`.
+
+## `this.resolve(context: string, request: string, callback: (err: Error | null, result: string) => void)`
```
Comment (chenjiahan) [inline website/docs/en/api/loader-api/context.mdx:74]: The title is too long, can we only show the function name in the title? like this:

````markdown
## this.resolve

- Type:

```ts
function resolve(
  context: string,
  request: string,
  callback: (err: Error | null, result: string) => void,
): void
```
````

File: website/docs/en/api/loader-api/context.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Resolve a request.
+
+- `context` must be the absolute path to a directory. This directory is used as the starting location for resolving.
+- `request` is the request to be resolved.
+- `callback` is a callback function that gives the resolved path.
+
+## `this.mode`
```
Comment (chenjiahan) [inline website/docs/en/api/loader-api/context.mdx:82]: The code block seems unnecessary in the title.

Remove the code block can be more clear:

- before:

## `this.mode`

- after:

## this.mode

**Comment by SyMind** [comment]
@chenjiahan done

**Review by chenjiahan — APPROVED** [review]
Awesome 👍

**Pull request merged**